### ISSUE TITLE
run_benchmarks doesn't work on Python 3

### ISSUE THREAD

**Issue by cdeil** [pbrod/numdifftools#5]
I wanted to execute the `run_benchmarks.py` for numdifftools.

There's this error on Python 3:
```
In [2]: import numdifftools.run_benchmark
N= 4
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-8fd62210f116> in <module>()
----> 1 import numdifftools.run_benchmark

/Users/deil/code/numdifftools/numdifftools/run_benchmark.py in <module>()
     49     ref_g = None
     50     f = BenchmarkFunction(N)
---> 51     for i, (key, Gradient) in enumerate(gradient_funs.iteritems()):
     52         t = time.time()
     53         gradient_f = Gradient(f)

AttributeError: 'OrderedDict' object has no attribute 'iteritems'
```

I'm not sure how I'm supposed to execute them ... usually examples and benchmarks are in a separate folder that is not installed with the Python package into site-packages.

Wouldn't that be better here, too, for the "main()" part that actually executes the code and produces a plot? The benchmark code and class could still be in the package.

cc @maniteja

**Comment by pbrod** [comment]
I usually run them from the eclipse ide I am using. This was something I
just threw together to see the performance of the different methods. I am
on my way to Italy now, so there will be a few days until I can do
something about it.

;-)
Per A.

**Comment by maniteja123** [comment]
Sorry for replying so late, couldn't find time to look at it. I think the error is because ``iteritems`` is deprecated in python3 for iterables like dict, OrderedDict etc.

**Comment by pbrod** [comment]
Resolved in version 0.9.11.
